website/source/docs/platform/k8s/helm.html.md

```diff
@@ -0,0 +1,455 @@
+---
+layout: "docs"
+page_title: "Helm - Kubernetes"
+sidebar_current: "docs-platform-k8s-helm"
+description: |-
+  The Vault Helm chart is the recommended way to install and configure Vault on Kubernetes.
+---
+
+# Helm Chart
+
+The [Vault Helm chart](https://github.com/hashicorp/vault-helm)
+is the recommended way to install and configure Vault on Kubernetes.
+In addition to running Vault itself, the Helm chart is the primary
+method for installing and configuring Vault to integrate with other
+services such as Consul for High Availability deployments.
+
+This page assumes general knowledge of [Helm](https://helm.sh/) and
+how to use it. Using Helm to install Vault will require that Helm is
+properly installed and configured with your Kubernetes cluster.
+
+-> **Important:** The Helm chart is new and
+may still change significantly over time. Please always run Helm with
+`--dry-run` before any install or upgrade to verify changes.
+
+~> **Security Warning:** By default, the chart will install an insecure configuration
+of Vault. This provides a less complicated out-of-box experience for new users,
+but is not appropriate for a production setup. It is highly recommended to use
+a properly secured Kubernetes cluster.
```

Comment:
Do we want to link to something like https://kubernetes.io/docs/tasks/administer-cluster/securing-a-cluster/ here?